Support additional GUNW job types (#110)
ASFHyP3/ingest-adapter: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [0.4.0]

### Added

> - Handle `INSAR_ISCE` and `ARIA_RAIDER` jobs in the same manner as `ARIA_S1_GUNW` jobs. Replaces the functionality of https://github.com/ASFHyP3/cloud-based-insar-operations

Author: ```suggestion
- Publish `INSAR_ISCE` and `ARIA_RAIDER` jobs to the `ARIA_S1_GUNW` CMR collection. Replaces the functionality of https://github.com/ASFHyP3/cloud-based-insar-operations
```
